src/com/akshay/captain/jira.main.kts

```diff
@@ -62,35 +69,105 @@ fun main() {
         tickets
             .filter { it.ticketType.isChange() }
             .ifNotEmpty { releaseNotes.add(changes) }
-            .map { releaseNotes.add("$changeEmoji ${it.key} \t ${it.summary}") }
+            .map { releaseNotes.add("$changeEmoji ${it.key} ${it.summary}") }
 
         // 3. Print all tech changes
         tickets
             .filter { it.ticketType is TicketType.Chapter }
             .ifNotEmpty { releaseNotes.add(techChanges) }
-            .map { releaseNotes.add("$techChangesEmoji ${it.key} \t ${it.summary}") }
+            .map { releaseNotes.add("$techChangesEmoji ${it.key} ${it.summary}") }
 
         // 4. Print all bug fixes
         tickets
             .filter { it.ticketType is TicketType.Bug }
             .ifNotEmpty { releaseNotes.add(bugFixes) }
-            .map { releaseNotes.add("$bugEmoji ${it.key} \t ${it.summary}") }
+            .map { releaseNotes.add("$bugEmoji ${it.key} ${it.summary}") }
 
         // 5. Print all instrumentation
         tickets
             .filter { it.ticketType is TicketType.Analytics }
             .ifNotEmpty { releaseNotes.add(instrumentation) }
-            .map { releaseNotes.add("$instrumentationEmoji ${it.key} \t ${it.summary}") }
+            .map { releaseNotes.add("$instrumentationEmoji ${it.key} ${it.summary}") }
 
-        val notes = releaseNotes.toList().joinToString(separator = "\n")
-        println(Gson().toJson(notes))
+        val newReleaseNotes = releaseNotes.toList().joinToString(prefix = "<p>", postfix = "</p>", separator = " <br/> ")
+        val newResult = newReleaseNotes.toList().joinToString(separator = "\n")
+        //6. Ask questions for announcement
+        println(newResult)
+        askQuestion(newResult, newReleaseNotes, false, webhookUrls, platform)
 
     } catch (exception: Exception) {
         println(exception.message)
 
     }
 }
 
+/**
+ * [askQuestion] is used to get user permission to start asking questions
+ */
+fun askQuestion(
+    newResult: String,
+    result: String,
+    isInputWrong: Boolean = false,
+    webhookUrls: List<Url>,
+    platform: Platform
+) {
+    when {
+        isInputWrong -> println("Please enter a correct input. Press Y/N")
```

Comment:
suggestion: remove press Y/N and show the actual error for the script. 
The usual flow should be to just throw the error with information and stop. The user should restart the process.

Reply:
But wouldn't that break the flow? Like redoing it again and again? I kept this to reduce the API calls

Reply:
AHHH is for the y/n answers!
thought it was a validation of the user input, you're right, let's keep it that way :)